java/test/src/main/resources/test_cross_language_invocation.py

```diff
@@ -9,14 +9,22 @@
 @ray.remote
 def py_func(value):
     assert isinstance(value, bytes)
-    return b"Response from Python: " + value
+    f = ray.java_function("org.ray.api.test.CrossLanguageInvocationTest",
+                          "BytesEcho")
+    r = f.remote(value)
+    return b"Response from Python: " + ray.get(r)
```

Comment:
```suggestion
    return b"Response from Java: " + ray.get(r)
```

Reply:
This message is for the first caller when Python call Java call Python.